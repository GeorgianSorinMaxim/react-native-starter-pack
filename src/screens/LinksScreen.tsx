import * as React from "react";
import { Linking, StyleSheet } from "react-native";
import { ScrollView } from "react-native-gesture-handler";

import LinkButton from "../components/LinkButton";

const LinksScreen = () => {
  return (
    <ScrollView
      style={styles.container}
      contentContainerStyle={styles.contentContainer}
    >
      <LinkButton
        label="Nando's"
        onPress={() => Linking.openURL("https://www.nandos.com")}
        isLastOption={false}
      />
    </ScrollView>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#fafafa"
  },
  contentContainer: {
    paddingTop: 15
  },
});

export default LinksScreen