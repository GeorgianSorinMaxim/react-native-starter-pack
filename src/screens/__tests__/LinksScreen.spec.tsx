import * as React from "react";
import renderer from "react-test-renderer";

import LinksScreen from "../LinksScreen";

jest.mock("@react-navigation/stack");
jest.mock("@react-navigation/native");
jest.mock("@react-navigation/bottom-tabs");

describe("LinksScreen", () => {
  it(`renders the LinksScreen screen`, () => {
    const tree = renderer.create(<LinksScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
