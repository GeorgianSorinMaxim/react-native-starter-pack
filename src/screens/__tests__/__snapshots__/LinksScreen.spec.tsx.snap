// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinksScreen renders the LinksScreen screen 1`] = `null`;
